src/CheckBox.css

```diff
@@ -0,0 +1,54 @@
+.genre-container {
```

Comment:
Separating checkbox into it's own css should help us keep it dynamic for both genre and platformers filtering.
Great jon